source/templates.tex

```diff
@@ -3367,9 +3367,8 @@
 \rSec3[temp.spec.partial.order]{Partial ordering of partial specializations}
 
 \pnum
-\indextext{more specialized!class template}%
 For two partial specializations,
-the first is \defn{more specialized} than the second if, given the following
+the first is \defnx{more specialized}{more specialized!class template} than the second if, given the following
```

Comment:
So, moving that under "class template" is not good; we should keep it generic, right?

Reply:
Function templates can't be partially specialized.

Reply:
The index entry already said `!class template` because the whole partial-specialization section used to assume it was a class template.  I'm just pointing out that a line you're touching is (and already was) outdated.

Function templates can be more specialized, but they're not relevant here.

Reply:
Fixed